[2024-06-19T10:00:00.321+0530] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-19T10:00:00.337+0530] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Backup_Database.backup_task scheduled__2024-06-19T04:20:00+00:00 [queued]>
[2024-06-19T10:00:00.342+0530] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Backup_Database.backup_task scheduled__2024-06-19T04:20:00+00:00 [queued]>
[2024-06-19T10:00:00.343+0530] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-06-19T10:00:00.358+0530] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): backup_task> on 2024-06-19 04:20:00+00:00
[2024-06-19T10:00:00.359+0530] {standard_task_runner.py:63} INFO - Started process 2904 to run task
[2024-06-19T10:00:00.362+0530] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'Backup_Database', 'backup_task', 'scheduled__2024-06-19T04:20:00+00:00', '--job-id', '100', '--raw', '--subdir', 'DAGS_FOLDER/basic.py', '--cfg-path', '/tmp/tmpc1q8z8qy']
[2024-06-19T10:00:00.364+0530] {standard_task_runner.py:91} INFO - Job 100: Subtask backup_task
[2024-06-19T10:00:00.407+0530] {task_command.py:426} INFO - Running <TaskInstance: Backup_Database.backup_task scheduled__2024-06-19T04:20:00+00:00 [running]> on host FN-21052948.
[2024-06-19T10:00:00.469+0530] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='sarika' AIRFLOW_CTX_DAG_ID='Backup_Database' AIRFLOW_CTX_TASK_ID='backup_task' AIRFLOW_CTX_EXECUTION_DATE='2024-06-19T04:20:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-06-19T04:20:00+00:00'
[2024-06-19T10:00:00.469+0530] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-19T10:00:00.477+0530] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-06-19T10:00:00.478+0530] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', '/home/kiit/airflow/bashScripts/backup.sh ']
[2024-06-19T10:00:00.483+0530] {subprocess.py:86} INFO - Output:
[2024-06-19T10:00:00.489+0530] {subprocess.py:93} INFO - mysql: [Warning] Using a password on the command line interface can be insecure.
[2024-06-19T10:00:00.494+0530] {subprocess.py:93} INFO - mysql: [Warning] Using a password on the command line interface can be insecure.
[2024-06-19T10:00:00.506+0530] {subprocess.py:93} INFO - mysqldump: [Warning] Using a password on the command line interface can be insecure.
[2024-06-19T10:00:00.510+0530] {subprocess.py:93} INFO - mysqldump: Error: 'Access denied; you need (at least one of) the PROCESS privilege(s) for this operation' when trying to dump tablespaces
[2024-06-19T10:00:00.655+0530] {subprocess.py:93} INFO - Backup completed in 0 seconds
[2024-06-19T10:00:00.655+0530] {subprocess.py:93} INFO - Backup Success in location: /home/kiit/airflow/backup | Required Space: 0GB, Available Space: 245.31GB
[2024-06-19T10:00:00.655+0530] {subprocess.py:97} INFO - Command exited with return code 0
[2024-06-19T10:00:00.655+0530] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-19T10:00:00.681+0530] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=Backup_Database, task_id=backup_task, run_id=scheduled__2024-06-19T04:20:00+00:00, execution_date=20240619T042000, start_date=20240619T043000, end_date=20240619T043000
[2024-06-19T10:00:00.735+0530] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-19T10:00:00.754+0530] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-06-19T10:00:00.761+0530] {local_task_job_runner.py:222} INFO - ::endgroup::
[2024-06-19T10:02:38.796+0530] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-19T10:02:38.812+0530] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Backup_Database.backup_task scheduled__2024-06-19T04:20:00+00:00 [queued]>
[2024-06-19T10:02:38.818+0530] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Backup_Database.backup_task scheduled__2024-06-19T04:20:00+00:00 [queued]>
[2024-06-19T10:02:38.818+0530] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-06-19T10:02:38.836+0530] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): backup_task> on 2024-06-19 04:20:00+00:00
[2024-06-19T10:02:38.838+0530] {standard_task_runner.py:63} INFO - Started process 3460 to run task
[2024-06-19T10:02:38.841+0530] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'Backup_Database', 'backup_task', 'scheduled__2024-06-19T04:20:00+00:00', '--job-id', '107', '--raw', '--subdir', 'DAGS_FOLDER/basic.py', '--cfg-path', '/tmp/tmpf2j14e2o']
[2024-06-19T10:02:38.842+0530] {standard_task_runner.py:91} INFO - Job 107: Subtask backup_task
[2024-06-19T10:02:38.896+0530] {task_command.py:426} INFO - Running <TaskInstance: Backup_Database.backup_task scheduled__2024-06-19T04:20:00+00:00 [running]> on host FN-21052948.
[2024-06-19T10:02:38.966+0530] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='sarika' AIRFLOW_CTX_DAG_ID='Backup_Database' AIRFLOW_CTX_TASK_ID='backup_task' AIRFLOW_CTX_EXECUTION_DATE='2024-06-19T04:20:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-06-19T04:20:00+00:00'
[2024-06-19T10:02:38.966+0530] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-19T10:02:38.974+0530] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-06-19T10:02:38.974+0530] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', '/home/kiit/airflow/bashScripts/backup.sh ']
[2024-06-19T10:02:38.979+0530] {subprocess.py:86} INFO - Output:
[2024-06-19T10:02:38.985+0530] {subprocess.py:93} INFO - mysql: [Warning] Using a password on the command line interface can be insecure.
[2024-06-19T10:02:38.990+0530] {subprocess.py:93} INFO - mysql: [Warning] Using a password on the command line interface can be insecure.
[2024-06-19T10:02:39.003+0530] {subprocess.py:93} INFO - mysqldump: [Warning] Using a password on the command line interface can be insecure.
[2024-06-19T10:02:39.170+0530] {subprocess.py:93} INFO - Backup completed in 0 seconds
[2024-06-19T10:02:39.170+0530] {subprocess.py:93} INFO - Backup Success in location: /home/kiit/airflow/backup | Required Space: 0GB, Available Space: 245.31GB
[2024-06-19T10:02:39.170+0530] {subprocess.py:97} INFO - Command exited with return code 0
[2024-06-19T10:02:39.170+0530] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-19T10:02:39.195+0530] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=Backup_Database, task_id=backup_task, run_id=scheduled__2024-06-19T04:20:00+00:00, execution_date=20240619T042000, start_date=20240619T043238, end_date=20240619T043239
[2024-06-19T10:02:39.214+0530] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-19T10:02:39.232+0530] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-06-19T10:02:39.241+0530] {local_task_job_runner.py:222} INFO - ::endgroup::
